### DISCUSSION TITLE
Cant access rtk query  use functions

### DISCUSSION THREAD

**Discussion by Alvi24** [reduxjs/redux-toolkit#3986] [Q&A]
export const productsApi = createApi({
  reducerPath: "productsApi",
  baseQuery: fetchBaseQuery({ baseUrl: env.VITE_BACKEND_URL }),
  endpoints: (builder) => ({
    getProducts: builder.query<Product[], void>({
      query: () => "products",
    }),
   
  }),
});

export const { useGetProductsQuery } = productsApi;  //Error: Property 'useGetProductsQuery' does not exist

**Comment by phryneas** [comment]
You need to import `createApi` from `/query/react`, not `/query`. Only the React entrypoint generates hooks for you.
